src/main.cpp

```diff
@@ -97,7 +97,7 @@ QCoreApplication* createApplication(int& argc, char* argv[])
             return new QCoreApplication(argc, argv);
         }
 #else
-#ifdef __linux__
+#if !(defined(_WIN32) || defined(__APPLE__))
```

Comment:
```suggestion
#if defined(__unix__)
```
That's more clear, I think. On https://erato1004.tistory.com/entry/Predefined-Compiler-Macros it states that macOS sets `BSD` but not `__unix__`.

Reply:
Sure. I thought MacOS set __unix__, but seems it doesn't.

Reply:
Although on the plus side the current option means we are only ever relying on the state of _WIN32, \_\_APPLE\_\_, and \_\_linux\_\_

Reply:
That's true. I wanted to change that because I initially misread it.

Reply:
Maybe we could change `!` to `not`? Shouldn't that work?

Reply:
Yeah, totally understand that. (Reordered some later ones for that exact same reason.) If you'd prefer \_\_unix\_\_ then I'm happy to go with that

Reply:
Yeah, totally understand that. (Reordered some later ones for that exact same reason.) If you'd prefer \_\_unix\_\_ then I'm happy to go with that.

Reply:
My main concern is the `!`. So I would be okay with `not` but I would prefer the `__unix__`.

Reply:
Let's go with \_\_unix\_\_ then.